backend/src/stwo/circuit_builder.rs

```diff
@@ -107,6 +107,12 @@ impl PowdrEval {
             .map(|(index, (_, id))| (id, index))
             .collect();
 
+        let publics_values = analyzed
```

Comment:
Lots of unrelated changes, I guess because my branch wasn't synced with main and yours was? I'll update mine